planet/System.c4g/LanguageDE.txt

```diff
@@ -875,14 +873,13 @@ IDS_MSG_NOMATS=Keine Materialdefinitionen verf
 IDS_MSG_NOMODIFYFAIRCREW=Faire Crew kann nicht ge�ndert werden: Durch Szenario vorgegeben!
 IDS_MSG_NOMODIFYORIGINAL=%s kann nicht aktualisiert werden, da es Teil eines Originalpakets ist.
 IDS_MSG_NOPARTICIPATE_DESC=%s von der n�chsten Runde ausnehmen.
-IDS_MSG_NOPLAYERSELECTED=Es sind keine Spieler ausgew�hlt. Bitte zuerst in der Spielerauswahl die Teilnehmer f�r diese Runde aktivieren.
+IDS_MSG_NOPLAYERSELECTED=Es sind keine Spieler ausgew�hlt. Bitte zuerst in der Spielerauswahl im Hauptmen� die Teilnehmer f�r diese Runde aktivieren.
```

Comment:
Please don't change the encoding.

Reply:
Resolved in IRC chat: GitHub bug, compare 
PR-Commit https://raw.githubusercontent.com/legacyclonk/LegacyClonk/afbbba9ed30db4e2d6bf9e634361a0237c4cab28/planet/System.c4g/LanguageDE.txt

Master https://raw.githubusercontent.com/legacyclonk/LegacyClonk/master/planet/System.c4g/LanguageDE.txt